game-core/src/main/java/games/strategy/engine/data/unit/ability/ConvertUnitAbility.java

```diff
@@ -0,0 +1,64 @@
+package games.strategy.engine.data.unit.ability;
+
+import games.strategy.engine.data.UnitType;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NonNull;
```

Comment:
nit, per previously established convention,  favor using `import javax.annotation.Nonnull;` over lombok 'NonNull'

https://github.com/triplea-game/triplea/issues?q=is%3Aissue+is%3Aclosed+NonNull+

Reply:
Done in https://github.com/triplea-game/triplea/pull/8646/commits/3ef07a4f3fe1e9ae6956a97806b33b0201254724